### DISCUSSION TITLE
Jam Tracks

### DISCUSSION THREAD

**Discussion by TexMexD** [soundshed/soundshed-app#32] [Q&A]
Can the JAM section find jam tracks on the local PC?  If so, how do you set the folder to search?  If not, could that be added?

**Comment by webprofusion-chrisc** [comment]
Hi, no it currently just loads videos from youtube. I can't totally see why this would be useful, maybe you could explain more?

It seems like you could just play them on your computer normally with the soundshed app still open and the only difference is that you'd be stopping/starting them with some other app.
